### ISSUE TITLE
[build] Undefined global variables in custom themes

### ISSUE THREAD

**Issue by Thanty** [fomantic/Fomantic-UI#2666]
# Bug Report

[This change](https://github.com/fomantic/Fomantic-UI/commit/0b65a824be9d1117ace8d3ce7f3e7652dc18b4f5#diff-f25c1f490ccd1c24a543a5c32115ea7731d2804dffdccf774933fca199dee435R19) broke my building process. I haven't override @site variable, so it has `default` value.

## Steps to reproduce
Let's focus on `divider` element for instance.

1. Create a new theme and use it in `theme.config`:
```
@divider: 'mytheme';
```
2. Override some variables inside this theme and use some of the global accessed (`/themes/mytheme/globals/site.variables`) variables (`/themes/mytheme/elements/divider.variables`):
```
@verticalDividerMargin: @gutterHorizontal * 2;
```
3. Run build process and get an error like this:

![Screenshot_6](https://user-images.githubusercontent.com/7223537/213414396-519cb0f3-abd9-40b2-b905-8e4533db2cba.png)

## Solution

![Screenshot_7](https://user-images.githubusercontent.com/7223537/213418304-9bdc216e-21ae-409b-9a72-08fde189b5d6.png)

## Version

2.9.1

**Comment by lubber-de** [comment]
Fixed by #2671

**Comment by Thanty** [comment]
Thanks, @lubber-de !
